neural_compressor/common/base_tuning.py

```diff
@@ -213,14 +213,21 @@ class TuningConfig:
         config_set: quantization configs. Default value is empty.
         timeout: Tuning timeout (seconds). Default value is 0 which means early stop.
         max_trials: Max tuning times. Default value is 100. Combine with timeout field to decide when to exit.
+        tolerable_loss: The metric loss is absolute, it can be both positive and negative. Default is 0.01.
```

Comment:
Recommend defining it `relative` to align with the usage in version 2.x.
https://github.com/intel/neural-compressor/blob/9ec22c96993d1a8dc00e0cebca1cbe5527ea8189/neural_compressor/config.py#L502-L510

This argument is crucial for users, so we should provide comprehensive documentation, including the computation formula and examples, to ensure clarity in its usage. Something like(subject to refinement):
```python
# example
tolerable_loss = 0.01
baseline = 100
eval_result_of_nth_trial = 99.1
eval_result_of_nth_trial > baseline * (1 - tolerable_loss ) , reached target.


```

Reply:
Sounds good!  When will we write the 3. x document or migration document?https://github.com/intel/neural-compressor/blob/master/docs/source/migration.md

Reply:
Let's include this information in the docstring first.